src/test/java/org/opentripplanner/standalone/server/TestRoutingService.java

```diff
@@ -0,0 +1,42 @@
+package org.opentripplanner.standalone.server;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.util.List;
+import org.opentripplanner.model.plan.Itinerary;
+import org.opentripplanner.model.plan.PlanTestConstants;
+import org.opentripplanner.model.plan.TripPlan;
+import org.opentripplanner.routing.api.RoutingService;
+import org.opentripplanner.routing.api.request.RouteRequest;
+import org.opentripplanner.routing.api.request.RouteViaRequest;
+import org.opentripplanner.routing.api.response.RoutingResponse;
+import org.opentripplanner.routing.api.response.ViaRoutingResponse;
+import org.opentripplanner.routing.framework.DebugTimingAggregator;
+
+public class TestRoutingService implements RoutingService {
```

Comment:
This is quite specific to the `GraphQLIntegrationTest`, I'd move it together in the same package